tests/package.lock

```diff
@@ -1,11 +1,17 @@
 prefixes:
-  host: host
-  tar: tar
-  ar: ar
+  ar: pkg-ar
+  host: pkg-host
+  tar: pkg-tar
 packages:
-  host:
-    path: ../packages/host
-  tar:
-    path: ../packages/tar
-  ar:
-    path: ../packages/ar
+  pkg-ar:
+    url: github.com/toitlang/pkg-ar
+    version: 1.0.0
+    hash: 175bcdfd59b2597c0825ef8765a20498a7e6a2db
+  pkg-host:
+    url: github.com/toitlang/pkg-host
+    version: 1.0.0
```

Comment:
This is now the wrong as the host-pkg has the wrong primitive interface.
Not completely sure how we want to fix this.